kubetest/main.go

```diff
@@ -715,10 +715,15 @@ func chmodArtifacts() error {
 }
 
 func prepare(o *options) error {
+	k8sProviderEnvVar := o.provider
+	if o.provider == "kubernetes-anywhere" {
+		// To make gce-related setup scripts in k8s to get sourced.
+		k8sProviderEnvVar = "gce"
```

Comment:
I can move this to another flag (sth like --kubernetes-anywhere-provider=gce). But like you said, maybe someone from test-infra should decide. cc @krzyzacy